test/test_theta_min_continuation.py

```
@@ -0,0 +1,111 @@
+Test script to verify theta_min-based continuation is working correctly
+"""
+
+import sys
+import os
+import numpy as np
+
+# Add parent directory to path
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from GLE_solver import solve_single_ca, find_critical_ca_lower_branch
+from GLE_continuation_hybrid import solve_bvp_for_ca, solve_for_theta_min_newton
```

Ruff (0.11.9)

`GLE_continuation_hybrid.solve_bvp_for_ca` imported but unused

Remove unused import: `GLE_continuation_hybrid.solve_bvp_for_ca`

(F401)